resources/tests/cpu_template_helper/fingerprint_AMD_MILAN_4.14host.json

```diff
@@ -1329,4 +1329,4 @@
       }
     ]
   }
-}
+}
```

Comment:
nit: here and below: no new line at the end of the file

Reply:
I don't put them in this PR, since the tool itself does not add those new lines and they were automatically generated by my editor in the past.